src/components/Form/UserResourceForm.vue

```diff
@@ -37,7 +37,7 @@ const selectedUser = ref("");
 const quantity = ref("");
 
 const userOptions = computed(() =>
-  store.getters["users/allUsers"].map((user) => user.name)
+  store.getters["users/allUsers"].map((user) => user.firstName)
```

Comment:
I think we should opt for email everywhere we'd need to search for a user.

Reply:
done